app/src/js/drawable/3d/grid3d.ts

```diff
@@ -63,26 +87,56 @@ export class Grid3D extends Shape3D {
     raycaster: THREE.Raycaster,
     intersects: THREE.Intersection[]
   ) {
-    if (this._control) {
-      this._control.raycast(raycaster, intersects)
+    if (this.label.selected || this.label.anyChildSelected()) {
+      const ray = raycaster.ray
+      const normal = new THREE.Vector3(0, 0, 1)
+      normal.applyEuler(this.rotation)
+      const plane = new THREE.Plane()
+      plane.setFromNormalAndCoplanarPoint(normal, this.position)
+      const target = new THREE.Vector3()
+      const intersection = ray.intersectPlane(plane, target)
+      if (intersection) {
+        // const worldToPlane = new THREE.Matrix4()
+        // worldToPlane.getInverse(this.matrixWorld)
```

Comment:
Is this comment needed for future code?

Reply:
Yes this may be needed in the future. We are also not using planes right now anyways.